### ISSUE TITLE
bug on v 4.1.17 - SyntaxError in node_modules\nativewind\node_modules\react-native-css-interop\.cache\web.css: Unexpected end of input   SyntaxError: Unexpected end of input

### ISSUE THREAD

**Issue by milon27** [nativewind/nativewind#1154]
with 4.1.16 it build for web. `npx expo export -p web` but its stuck on mobile build `npx eas build --platform android --profile development`

with 4.1.17 it build for mobile build `npx eas build --platform android --profile development`.  but its give error on `npx expo export -p web` 

error: 

```bash
> expo export -p web

Using src/app as the root directory for Expo Router.
Starting Metro Bundler
Web E:\Milon_Project\JS_PROJECTS\client\_m27lab\_mine\password-manager\password-manager-frontend\node_modules\expo-router/entry.js ▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓░ 
99.9% (1356/1357)SyntaxError: SyntaxError in node_modules\nativewind\node_modules\react-native-css-interop\.cache\web.css: Unexpected end of input  
SyntaxError: Unexpected end of input
    at transform (E:\Milon_Project\JS_PROJECTS\client\_m27lab\_mine\password-manager\password-manager-frontend\node_modules\@expo\metro-config\build\transform-worker\transform-worker.js:131:24)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
```

**Comment by GustavoBonfimS** [comment]
Same here when exporting for web

**Comment by marklawlor** [comment]
Fixed in `4.1.19`
